lxd/storage/pool_interface.go

```diff
@@ -80,7 +80,7 @@ type Pool interface {
 
 	// Custom volumes.
 	CreateCustomVolume(projectName string, volName string, desc string, config map[string]string, contentType drivers.ContentType, op *operations.Operation) error
-	CreateCustomVolumeFromCopy(projectName string, volName, desc string, config map[string]string, srcPoolName, srcVolName string, srcVolOnly bool, op *operations.Operation) error
+	CreateCustomVolumeFromCopy(projectName string, targetProjectName string, volName, desc string, config map[string]string, srcPoolName, srcVolName string, srcVolOnly bool, op *operations.Operation) error
```

Comment:
For those two, I think we want to reverse it. The first argument is the target projectName, the second should be the source project name. So probably something like `srcProjectName` instead?